base_search_mail_content/models/mail_thread.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from lxml import etree
+from openerp.osv import expression
+from openerp.osv.orm import setup_modifiers
```

Comment:
@dreispt The method is still there [setup_modifiers in v9](https://github.com/odoo/odoo/blob/9.0/openerp/osv/orm.py#L88)
And I cannot find any alternative for this. I guess we have to avoid this warning.